recipes/phytop/meta.yaml

```
@@ -0,0 +1,159 @@
+    - python >=3.8
+    - biopython
+    - matplotlib
+    - networkx
+    - lazy-property
+    - ete3
+    - numpy
+    - scipy
+    
+about:
+  home: https://github.com/zhangrengang/phytop/
+  description: "Phytop: A tool for visualizing and recognizing signals of incomplete lineage sorting and hybridization using species trees output from ASTRAL"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)